src/components/LanguageSection.jsx

```diff
@@ -35,9 +36,6 @@ const LanguageSection = props => {
         }}
         onChange={sel => onChange(fieldName, sel.value)}
       />
-      <Typography variant="body1" component="div">
-        <ReactMarkdownWrapper source={t('ProfileView.locale.contrib')} />
-      </Typography>
```

Comment:
The imports should be removed

Reply:
Linter should have raised an error?

Reply:
@Crash-- Absolutely, we discussed about that with @Noemie21 and now the IDE will show such errors ;)